setup.cfg

```diff
@@ -101,8 +101,17 @@ filterwarnings =
     ignore:the imp module is deprecated in favour of importlib:DeprecationWarning:ipykernel.iostream
     # Ignore warnings about asdf versions
     ignore:File.*asdf.extension.BuiltinExtension.*:asdf.exceptions.AsdfWarning
+    # Ignore asdf complaining about files written with old extension
+    ignore:File.*was created with extension class .*DKISTExtension.* which is not currently installed: asdf.exceptions.AsdfWarning
```

Comment:
There's a feature of the new-style extension that helps with this:

https://github.com/asdf-format/asdf/blob/master/asdf/extension/_extension.py#L39